extensions/eclipse-che-theia-plugin-ext/package.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "@eclipse-che/theia-plugin-ext",
+    "version": "1.0.0",
+    "description": "Theia extension",
+    "author": "Yevhen Vydolob",
+    "license": "EPL-2.0",
+    "keywords": [
+        "theia-extension",
+        "che"
+    ],
+    "files": [
+        "lib",
+        "src"
+    ],
+    "dependencies": {
+        "@theia/core": "next",
```

Comment:
I'm not sure it can be next there for CQs

Reply:
Theia soon will be Eclipse project, but OK lets fix current version

Reply:
Is it OK to keep `next` version till `0.3.17` released @benoitf ?

Reply:
yes sure

Reply:
0.3.17 has been released